src/js/post-911-gib-status/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+import moment from 'moment';
+
+export function getDateFormatted(date) {
```

Comment:
These can probably be called simply `formatDate()` and `formatPercent()` since we aren't actually getting the date or percent value from anywhere.